src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -0,0 +1,20 @@
+namespace System.Security
+{
+    public abstract partial class CodeAccessPermission : System.Security.IPermission, System.Security.ISecurityEncodable, System.Security.IStackWalk
+    {
+        protected CodeAccessPermission() { }
+        public void Assert() { }
+        public abstract System.Security.IPermission Copy();
+        public void Demand() { }
+        public void Deny() { }
+        public override bool Equals(object obj) { return default(bool); }
+        //    public abstract void FromXml(System.Security.SecurityElement elem);
```

Comment:
Is adding members that aren't coming back in this change as commented out par for the course here? Our normal rule is no commented-out code (unless it's an example).